scripts/build-transmission.js

```diff
@@ -66,15 +66,16 @@ const build = async () => {
     const flags = [
       ...COMMON_CMAKE_FLAGS,
       // Set vcpkg toolchain file path
-      `-DCMAKE_TOOLCHAIN_FILE=${VCPKG_INSTALLATION_ROOT}\\scripts\\buildsystems\\vcpkg.cmake`,
+      `-DCMAKE_TOOLCHAIN_FILE=${VCPKG_INSTALLATION_ROOT}/scripts/buildsystems/vcpkg.cmake`,
       // Set include and lib dir paths to static libcurl
-      `-DCURL_INCLUDE_DIR=${VCPKG_INSTALLATION_ROOT}\\packages/curl_x64-windows-static/include`,
-      `-DCURL_LIBRARY=${VCPKG_INSTALLATION_ROOT}\\packages/curl_x64-windows-static/lib`,
+      `-DCURL_INCLUDE_DIR=${VCPKG_INSTALLATION_ROOT}/packages/curl_x64-windows-static/include`,
+      `-DCURL_LIBRARY=${VCPKG_INSTALLATION_ROOT}/packages/curl_x64-windows-static/lib/libcurl.lib`,
+      `-DOPENSSL_ROOT_DIR=${VCPKG_INSTALLATION_ROOT}/packages/openssl_x64-windows-static`,
       // Use static version of the run-time library
-      '-DCMAKE_C_FLAGS=/MT',
-      '-DCMAKE_CXX_FLAGS=/MT',
-      '-DCMAKE_C_FLAGS_RELEASE=/MT',
-      '-DCMAKE_CXX_FLAGS_RELEASE=/MT'
+      '-DCMAKE_C_FLAGS=/MT /MP',
+      '-DCMAKE_CXX_FLAGS=/MT /MP',
+      '-DCMAKE_C_FLAGS_RELEASE=/MT /MP',
+      '-DCMAKE_CXX_FLAGS_RELEASE=/MT /MP'
     ]
```

Comment:
Not sure to understand this change?

Reply:
a flag to build with `msvc` with multiple processes 🚀 
I thought cmake was already settings this flags as we added the `j` flag, but it seems it is not.

https://learn.microsoft.com/en-us/cpp/build/reference/mp-build-with-multiple-processes?view=msvc-170

Reply:
Maybe we should highlight to the cmake community? https://discourse.cmake.org/

Reply:
It seems the issue is already known https://discourse.cmake.org/t/parallel-does-not-really-enable-parallel-compiles-with-msbuild/964